ui/page/staking/stake_overview.go

```diff
@@ -46,7 +47,8 @@ type Page struct {
 	*app.GenericPageModal
 	*listeners.TxAndBlockNotificationListener
 
-	scroll *components.Scroll[*transactionItem]
+	scroll          *components.Scroll[*transactionItem]
+	scrollContainer *widget.List
```

Comment:
Not related to this PR but out of curiosity, why do we usually combine a `components.Scroll` and a `widget.List`? They both use `layout.List` underneath and we eventually have a `layout.List` inside another `layout.List` when it looks like we can have just one. I'll probably dig into this code myself later.

Reply:
agree with use, may be we need optimize this component but in other PR